spec/e2e/accessibility_spec.js

```diff
@@ -25,36 +23,40 @@ describe('accessibility', () => {
 
   for (const [label, viewport] of Object.entries(viewports)) {
     describe(`${label} viewport`, () => {
-      test.concurrent.each(paths)('%s', async (path) => {
-        const page = await global.browser.newPage();
-        await page.setViewport(viewport);
-        await page.goto(new URL(path, process.env.ROOT_URL).toString());
-        const runner = new AxePuppeteer(page).disableRules('frame-tested').disableFrame('*');
-
-        const results = await runner.analyze();
-        expect(results).toHaveNoViolations();
-
-        const links = await getCandidateLinks(page);
-        links.forEach((a) => expect(a).toNotHaveTargetBlank());
-        await page.close();
+      paths.forEach((path) => {
+        test(path, async () => {
+          const page = await global.browser.newPage();
+          await page.setViewport(viewport);
+          await page.goto(new URL(path, process.env.ROOT_URL).toString());
+          const runner = new AxePuppeteer(page).disableRules('frame-tested').disableFrame('*');
+
+          const results = await runner.analyze();
+          assert.deepEqual(results.violations, []);
+
+          const links = await getCandidateLinks(page);
+          links.forEach((a) => assert.notEqual(a.target, '_blank'));
+          await page.close();
+        });
       });
     });
   }
 
   describe('"Back to top" link', () => {
     it('resets focus to the beginning of content', async () => {
-      await goto('/about-us/');
-      const backToTopLinkHandle = await globalPage.$('.usa-prose .anchor-to-top');
+      const page = await global.browser.newPage();
+      await page.goto(getURL('/about-us/'));
+      const backToTopLinkHandle = await page.$('.usa-prose .anchor-to-top');
       await backToTopLinkHandle.click();
-      await globalPage.keyboard.press('Tab');
-      const isFocusBeforeBackToTop = await globalPage.evaluate(
+      await page.keyboard.press('Tab');
```

Comment:
I don't love losing the `goto` helper. Considering a future pull request could create some helpers to abstract this a bit more, maybe in the form of the [IdP's test lifecycle helpers](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/test-helpers), e.g.

```ts
function useBrowser() {
  let pages = [];
  afterEach(async () =>
    Promise.all(pages.map((page) => page.close()));
  );

  async function createPage() {
    const page = await global.browser.newPage();
    pages.push(page);
    const goto = (path) => page.goto(getURL(path));
    return { goto };
  };

  return { createPage };
}
```

```ts
describe('something', () => {
  const { createPage } = useBrowser();

  it('does a thing', () => {
    createPage().goto('/about-us/');
  });
});
```

Reply:
what about the goto helper makes us want to avoid it?

Reply:
I edited my comment shortly after posting it which includes a bit more detail about why it's necessary:

>it's needed in order to support concurrency, since we can't just have a single page that all concurrent tests try to use at the same time.